### DISCUSSION TITLE
Tasks managing

### DISCUSSION THREAD

**Discussion by rafacouto** [rust-lang-ve/hilow#6] [General]
It'd be convenient to set _milestones_ with goals and manage a Kanban board to assign tasks to members. What do you think about?

**Comment by LeoBorai** [comment]
Totally, I'm sure @JMEspiz agree with this as well.

I think we could take advantage of Automated Kanban Boards from GitHub for this.

I would like to be able to have all of our project management here on GitHub as well, from issues to projects (kanban boards) and milestones.

With the endpoints  definitions in place we can move on to the definition of requirements and then define todos to achieve these as well.

**Reply by JMEspiz** [reply]
Absolutely agree. I've not used Github since six month ago, but there wasn't a Project option for managing task assignation?
